src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractRazorEditorTest.cs

```diff
@@ -192,6 +194,19 @@ static void WaitForFileExistsAsync(string file)
             }
         }
 
+        private static void EnsureExtensionInstalled()
+        {
+            var assembly = Assembly.Load(new AssemblyName("Microsoft.VisualStudio.RazorExtension"));
+            var version = assembly.GetName().Version;
+
+            var localAppData = Environment.GetEnvironmentVariable("LocalAppData");
+
+            if (!version.Equals(new Version(42, 42, 42, 42)) || !assembly.Location.StartsWith(localAppData, StringComparison.OrdinalIgnoreCase))
+            {
+                throw new NotImplementedException("Integration test not running against Experimental Extension");
```

Comment:
Nit: Is this true? Or is it that it is running against the experimental instance, but the extension didn't deploy correctly?

Reply:
Hmm, I think the way I phrased it is technically accurate (in this case we don't have the Experimental Extension, just the Experimental Instance), but I can see how the phrasing is confusing. Do you have a suggestion on making it clearer?

Reply:
> we don't have the Experimental Extension, just the Experimental Instance

Thank you. I misread the last word of the message.